test/utils.throttle.js

```diff
@@ -1,25 +1,37 @@
 const requestLib = require('got');
 const throttled = require('../lib/utils/throttle');
+const sinon = require('sinon');
+const assert = require('chai').assert;
 
-it('Should make three requests with 5000ms interval. (Throttle function)', function (done) {
-  this.timeout(30000);
-  const req = throttled(requestLib, {
-    limit: 1,
-    interval: 5000
+describe('Throttle tests', function () {
+  let server;
+  this.timeout(15000);
```

Comment:
The throttle test used to call external api. If we don't want to rely on the availability of this particular api we can use mocks.
The fake server intercept http calls and return specified objects if it mach the same method/url.

Reply:
It could be extended to play store calls, but if we mocks responses we won't be able to see play store interface changes.

Reply:
I think it's better to keep it agnostic since the module is generic as well (ideally we wouldn't implement this inside this package, instead rely on another module #650 but I think at some point this was introduced because of a change in the requests library)

Reply:
Yes, i also searched for throttle libraries, but i found libraries that were unable to handle both interval and number of requests. I'll try the p-trottle you mentionned.

Reply:
p-throttle needs esm migration. i'll try it after esm migration.